pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -39,7 +42,7 @@ type MercuryTriggerService struct {
 var _ services.Service = &MercuryTriggerService{}
 
 type subscriberConfig struct {
-	FeedIds        []string
+	FeedIDs        []string
```

Comment:
@ettec I think you're missing a JSON tag here -- this is populated from line 102, and the incoming data will use `FeedIds` as the key.

Reply:
I've reverted this rename - was showing up as an issue when I locally linted but seems fine on CI